### PR TITLE
Remove liquibase listener

### PR DESCRIPTION
**Pull request by rayaviri**
Remove the liquibase listener

### CODE REVIEW

**Review by Ramir0 — APPROVED** [review]

File: src/main/webapp/WEB-INF/web.xml [hunk 1]
```diff
@@ -1,39 +1,36 @@
...
-    <url-pattern>/action/*</url-pattern>
-  </servlet-mapping>
-  <resource-ref>
-    <description>DB Connection</description>
-    <res-ref-name>efeeder</res-ref-name>
-    <res-type>javax.sql.DataSource</res-type>
-    <res-auth>Container</res-auth>
-  </resource-ref>
+	<context-param>
+		<param-name>liquibase.changelog</param-name>
```
Comment (ziur) [inline src/main/webapp/WEB-INF/web.xml]: These parameters are not needed anymore due to the listener was removed.

File: src/main/webapp/WEB-INF/web.xml [hunk 2]
```diff
@@ -1,39 +1,36 @@
...
+	</context-param>
+	<context-param>
+		<param-name>liquibase.datasource</param-name>
+		<param-value>java:comp/env/jdbc/efeeder</param-value>
+	</context-param>
+	<listener>
+		<listener-class>org.jala.efeeder.servlets.ApplicationContextListener</listener-class>
+	</listener>
+	<servlet>
+		<servlet-name>GreetingServlet</servlet-name>
```
Comment (ziur) [inline src/main/webapp/WEB-INF/web.xml]: And could you please remove this servlet :D?

**Review by ziur — CHANGES_REQUESTED** [review]
Please, remove context params related to liquidbase

File: src/main/webapp/WEB-INF/web.xml [hunk 3]
```diff
@@ -1,39 +1,24 @@
...
-    <url-pattern>/action/*</url-pattern>
-  </servlet-mapping>
-  <resource-ref>
-    <description>DB Connection</description>
-    <res-ref-name>efeeder</res-ref-name>
-    <res-type>javax.sql.DataSource</res-type>
-    <res-auth>Container</res-auth>
-  </resource-ref>
+	<context-param>
+		<param-name>liquibase.datasource</param-name>
```
Comment (ziur) [inline src/main/webapp/WEB-INF/web.xml]: This one is not required too

**Review by ziur — APPROVED** [review]

**Pull request merged**